Doc/library/re.rst

```diff
@@ -1586,6 +1590,7 @@ beginning of the string, whereas using :func:`search` with a regular expression
 beginning with ``'^'`` will match at the beginning of each line. ::
 
    >>> re.match('X', 'A\nB\nX', re.MULTILINE)  # No match
```

Comment:
For consistency, other examples seem to use double quotes so this and `re.search` can be swapped over, or `re.fullmatch` can be changed to single quote (but optional).

Reply:
Done the changes